src/main/java/de/tum/in/www1/artemis/config/StaticCodeAnalysisConfigurer.java

```diff
@@ -30,8 +31,12 @@ public StaticCodeAnalysisConfigurer() {
 
     @PostConstruct
     private void init() {
-        languageToDefaultCategories = Map.of(ProgrammingLanguage.JAVA, createDefaultCategoriesForJava(), ProgrammingLanguage.SWIFT, createDefaultCategoriesForSwift());
-        log.debug("Initialized default static code analysis categories for JAVA");
+        languageToDefaultCategories = Map.of(ProgrammingLanguage.JAVA, createDefaultCategoriesForJava(), ProgrammingLanguage.SWIFT, createDefaultCategoriesForSwift(),
+                ProgrammingLanguage.C, createDefaultCategoriesForC());
+
+        List<ProgrammingLanguage> supportedLanguages = new LinkedList<>(languageToDefaultCategories.keySet());
+
+        log.debug("Initialized default static code analysis categories for: " + supportedLanguages);
```

Comment:
```suggestion
        log.debug("Initialized default static code analysis categories for: {}", languageToDefaultCategories.keySet());
```
Using the keySet directly should work as well.

Reply:
I like it, as it is a bit shorter. Having an extra variable would be a little bit more descriptive, but I think that is not needed here.